scripts/library.sh

```diff
@@ -23,7 +23,13 @@ readonly SERVING_GKE_VERSION=gke-latest
 readonly SERVING_GKE_IMAGE=cos
 
 # Public latest stable nightly images and yaml files.
-readonly KNATIVE_BASE_YAML_SOURCE=https://storage.googleapis.com/knative-nightly/@/latest
+KNATIVE_BASE_YAML_SOURCE=""
```

Comment:
Move any code/condition dependent initialization to the end of the file, together with the other vars.

Reply:
Done.